notebooks/official/training/pytorch_gcs_data_training.ipynb

```diff
@@ -0,0 +1,1101 @@
+{
```

Comment:
<p>Rewrite this in present tense: For this tutorial, a CSV version of the dataset available on <a href="https://www.kaggle.com/datasets/oddrationale/mnist-in-csv?select=mnist_train.csv" target="_blank">Kaggle</a> has been uploaded to a Cloud Storage bucket for your use.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1705/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
<p>(The future tense is what's to be avoided. See here: https://developers.google.com/style/tense)</p>